Sources/Swift/Protocol/SentrySdkInfo.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+
+@objc
+enum SentrySdkPackageManager: Int {
+    case spm
+    case cocoapods
+    case carthage
+    case unknown
+}
+
+@objcMembers
+class SentrySdkInfo: NSObject, SentrySerializable {
+    /**
+     * This is required to identify the package manager used when installing sentry.
+     */
+#if SWIFT_PACKAGE
+    static let PACKAGE_MANAGER: SentrySdkPackageManager = .spm
+#elseif COCOAPODS
+    static let PACKAGE_MANAGER: SentrySdkPackageManager = .cocoapods
+#elseif CARTHAGE_YES
+    // CARTHAGE is a xcodebuild build setting with value `YES`, we need to convert it into a compiler
+    // definition to be able to use it.
+    static let PACKAGE_MANAGER: SentrySdkPackageManager = .carthage
+#else
+    static let PACKAGE_MANAGER: SentrySdkPackageManager = .unknown
+#endif
```

Comment:
@brustolin @philipphofmann What is the easiest way to check that these compiler directives work also in Swift?

Reply:
I see for example SWIFT_PACKAGE we already use in swift so that should work.